<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Многослойный перцептрон + генетический алгоритм</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_dataset">
     <attribute name="title">
      <string>Весь датасет</string>
     </attribute>
     <widget class="QPushButton" name="button_view_dataset">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Открыть в отдельном окне</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_fit">
     <attribute name="title">
      <string>Обучение</string>
     </attribute>
     <widget class="QPushButton" name="button_run_fit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Запустить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_view_fit">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Открыть в отдельном окне</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Поколение</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Лучшая точность</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Средняя точность</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Среднеквадратичное отклонение</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_best_acc">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>80</y>
        <width>59</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_generation">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>60</y>
        <width>59</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_mean_acc">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>107</y>
        <width>59</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_deviation">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>130</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_test">
     <attribute name="title">
      <string>Тестирование</string>
     </attribute>
     <widget class="QPushButton" name="button_run_test">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Запустить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_view_test">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Открыть в отдельном окне</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>50</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Точность</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_acc">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="double_spin_box_spread">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>65</width>
        <height>27</height>
       </rect>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Величина спреда</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Доходность</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_profit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>120</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Датасет</string>
    </property>
    <addaction name="action_load_dataset"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Конфигурация</string>
    </property>
    <addaction name="action_load_conf"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Популяция</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action_2">
   <property name="text">
    <string>Загрузить</string>
   </property>
  </action>
  <action name="action_load_conf">
   <property name="text">
    <string>Загрузить</string>
   </property>
  </action>
  <action name="action_load_dataset">
   <property name="text">
    <string>Загрузить</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Создать новую</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Сохранить как</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Загрузить</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
